### PR TITLE
Update product name to `myWitWallet`

### LINKED ISSUE #126: Rename wallet to `mywitwallet`
We need to make an effort to be consistent with the naming and to push the `mywitwallet` branding. As part of that, one step is to rename the repo.

### PR DESCRIPTION
**Pull request by gabaldon**
- Updates product name to `myWitWallet`

close #126

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# witnet_wallet
+# myWitWallet
```
Comment (aesedepece) [inline README.md:1]: @witnet/devs @witnet/brick-mortars `MyWitWallet` or `myWitWallet`?

Comment (gabaldon) [reply README.md:1]: I prefer `myWitWallet`

Comment (aesedepece) [reply README.md:1]: I think I prefer `myWitWallet` too. It does bring more attention to the `Wit` part.

Comment (Tommytrg) [reply README.md]: I prefer `MyWitWallet` because it seems rare to me that a proper noun doesn't start with a capital letter.

**Review by Tommytrg — APPROVED** [review]

**Pull request merged**